From 683e6b3e4d3b43b03e819726eba2813b613d2d02 Mon Sep 17 00:00:00 2001
From: Huang Shan <huang.shan@freescale.com>
Date: Tue, 29 Apr 2014 16:12:59 +0800
Subject: [PATCH 6/6] ENGR00293718  No other language display for Show system
 time when change to other UI language.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

add items
<string name="show_system_time">"显示系统时间"</string>
<string name="show_system_time_summary">"屏幕叠加显示系统当前时间"</string>
Settings/res/values-zh-rCN/strings.xml b/res/values-zh-rCN/strings.xml

Signed-off-by: Huang Shan <huang.shan@freescale.com>
---
 res/values-zh-rCN/strings.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/res/values-zh-rCN/strings.xml b/res/values-zh-rCN/strings.xml
index fdca466..ae5cd50 100644
--- a/res/values-zh-rCN/strings.xml
+++ b/res/values-zh-rCN/strings.xml
@@ -2042,4 +2042,6 @@
     <string name="user_image_take_photo" msgid="1280274310152803669">"拍照"</string>
     <string name="user_image_choose_photo" msgid="7940990613897477057">"从图库中选择照片"</string>
     <string name="regulatory_info_text" msgid="5623087902354026557"></string>
+    <string name="show_system_time">"显示系统时间"</string>
+    <string name="show_system_time_summary">"屏幕叠加显示系统当前时间"</string>
 </resources>
-- 
1.8.0

